### ISSUE TITLE
Migrating from dbus-glib to GDbus

### ISSUE THREAD

**Issue by GoogleCodeExporter** [yendo/gphotoframe#103]
```
Migrating to GDBus
http://developer.gnome.org/gio/2.30/ch29.html
```

Original issue reported on code.google.com by `yendo0206` on 21 Dec 2011 at 4:13

**Comment by GoogleCodeExporter** [comment]
```
Gio.bus_get_sync - http://www.piware.de/2011/01/na-zdravi-pygi/
```

Original comment by `yendo0206` on 27 Dec 2011 at 7:28
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Implemented in revision 840d17633fe9.
```

Original comment by `yendo0206` on 15 Feb 2012 at 3:24
- Changed state: **Fixed**
